spinalcordtoolbox/deepseg/inference.py

```diff
@@ -238,6 +238,9 @@ def segment_nnunet(path_img, tmpdir, predictor, device: torch.device):
         model_orientation = "RPI"
     elif "RegionBasedLesionSeg" in predictor.plans_manager.dataset_name:
         model_orientation = "AIL"
+    elif "msLesionAgnostic" in predictor.plans_manager.dataset_name:
+        # For model_seg_ms_lesion in release r20241101: https://github.com/ivadomed/ms-lesion-agnostic/releases/tag/r20241101
+        model_orientation = "RPI"
```

Comment:
It doesn't change much in terms of output, since Left-right or Right-Left for spinal cord is rather similar. However, because I trained my model on RPI, I still changed this parameter. 
Moreover, in some context this parameter is very important and can be easily forgotten since it is not stored with the models. Maybe adding "orientation":"RPI" in the models.py could make sense in the future. What do you think @joshuacwnewton ?

Reply:
just FYI: we were discussing an idea to include the orientation to `dataset.json` as a key-value pair, for example: `"orientation": "LPI"`. Details: https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/4654

Reply:
Ok, so it seems that this is the way to go now. Are we sure that it doesn't have an impact on model training or inference?

Reply:
After discussion during the meeting we confirmed that the we should add this information in the dataset.json